components/provisioner/third_party/machinebox/graphql/README.md

```diff
@@ -0,0 +1,67 @@
+# graphql [![GoDoc](https://godoc.org/github.com/machinebox/graphql?status.png)](http://godoc.org/github.com/machinebox/graphql) [![Build Status](https://travis-ci.org/machinebox/graphql.svg?branch=master)](https://travis-ci.org/machinebox/graphql) [![Go Report Card](https://goreportcard.com/badge/github.com/machinebox/graphql)](https://goreportcard.com/report/github.com/machinebox/graphql)
+
+Low-level GraphQL client for Go.
+
+* Simple, familiar API
+* Respects `context.Context` timeouts and cancellation
+* Build and execute any kind of GraphQL request
+* Use strong Go types for response data
+* Use variables and upload files
+* Simple error handling
+
+## Installation
+Make sure you have a working Go environment. To install graphql, simply run:
+
+```
+$ go get github.com/machinebox/graphql
+```
+
+## Usage
+
```

Comment:
Please add an introductory sentence describing what to do with the code below.

Reply:
Hey! I don't think I should change it. It's just a copied library from the [machinebox repository](https://github.com/machinebox/graphql). The only change I made is a minor contribution which allows us to use it in our specific way

Reply:
I see. In that case, please make sure that the appropriate credits and sources are included (see point 4 in the license). 
I think a note in the form of a comment could also be added in modified parts of code. For example, in `client.go` where we have line 9 (`"github.com/kyma-project/control-plane/components/provisioner/third_party/machinebox/graphql"`), we could add a comment saying something like "The original code comes from `https://github.com/machinebox/graphql`, however, this line was modified." or something like this.